### PR TITLE
Adapt the quickstart image

### PR DESCRIPTION
**Pull request by frascuchon**
This PR makes several modifications to the Argilla quickstart image:

- Replaces "Team" and "Argilla" users with "Admin" and "Annotator" users.
- Aligns all environment variables with their corresponding user types: ADMIN_* and ANNOTATOR_*.
- For admin users, it is possible to set up the username, API key, and password using the ADMIN_USERNAME, ADMIN_API_KEY, and ADMIN_PASSWORD variables.
- For annotator users, it is possible to set up the username and password using the ANNOTATOR_USERNAME and ANNOTATOR_PASSWORD variables.
- By default, the operational workspace will be based on the annotator's username. However, this can be overridden using the ARGILLA_WORKSPACE environment variable.

### CODE REVIEW

File: quickstart.README.md [hunk 1]
```diff
@@ -113,16 +113,18 @@ This will run the latest quickstart docker image with 2 users `argilla` and `tea
 
 ### Environment Variables
 
-- `ARGILLA_API_KEY`: Argilla provides a Python library to interact with the app (read, write, and update data, log model
+- `ADMIN_USERNAME`: The admin username to login in Argilla. The default admin username is `admin`. By setting up
+  a custom username you can use your own username to login into the app.
+- `ADMIN_API_KEY`: Argilla provides a Python library to interact with the app (read, write, and update data, log model
   predictions, etc.). If you don't set this variable, the library and your app will use the default API key
-  i.e. `argilla.apikey`. If you want to secure your app for reading and writing data, we recommend you to set up this
+  i.e. `admin.apikey`. If you want to secure your app for reading and writing data, we recommend you to set up this
```
Comment (frascuchon) [inline quickstart.README.md:120]:
```suggestion
  i.e. `team.apikey`. If you want to secure your app for reading and writing data, we recommend you to set up this
```

File: quickstart.Dockerfile [hunk 2]
```diff
@@ -1,44 +1,44 @@
 FROM docker.elastic.co/elasticsearch/elasticsearch:8.5.3
 
 # Environment variable
-ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/usr/share/elasticsearch/users.yml
-ENV TEAM_PASSWORD=1234
-ENV ARGILLA_PASSWORD=1234
-ENV TEAM_API_KEY=team.apikey
-ENV ARGILLA_API_KEY=argilla.apikey
+ENV ADMIN_USERNAME=admin
+ENV ADMIN_PASSWORD=11223344
```
Comment (jfcalvo) [inline quickstart.Dockerfile]: I think this one is more simple and easy to remember:

```suggestion
ENV ADMIN_PASSWORD=12345678
```

File: quickstart.Dockerfile [hunk 3]
```diff
@@ -1,44 +1,44 @@
...
+ENV ADMIN_API_KEY=team.apikey
+
+ENV ANNOTATOR_USERNAME=argilla
+ENV ANNOTATOR_PASSWORD=11223344
```
Comment (jfcalvo) [inline quickstart.Dockerfile]:
```suggestion
ENV ANNOTATOR_PASSWORD=12345678
```

File: quickstart.Dockerfile [hunk 4]
```diff
@@ -1,44 +1,44 @@
...
 
 # Environment variable
-ENV ARGILLA_LOCAL_AUTH_USERS_DB_FILE=/usr/share/elasticsearch/users.yml
-ENV TEAM_PASSWORD=1234
-ENV ARGILLA_PASSWORD=1234
-ENV TEAM_API_KEY=team.apikey
-ENV ARGILLA_API_KEY=argilla.apikey
+ENV ADMIN_USERNAME=admin
+ENV ADMIN_PASSWORD=11223344
+ENV ADMIN_API_KEY=team.apikey
```
Comment (jfcalvo) [inline quickstart.Dockerfile]: Now that we are changing everything from team to admin this make sense, right?:

```suggestion
ENV ADMIN_API_KEY=admin.apikey
```

File: quickstart.Dockerfile [hunk 5]
```diff
@@ -1,44 +1,44 @@
...
-
-COPY scripts/start_quickstart_argilla.sh /
-COPY scripts/load_data.py /
-COPY dist/*.whl /packages/
-
-# Install argilla
-RUN chmod +x /start_quickstart_argilla.sh && \
+    pip3 install datasets \
+    # Install Argilla
+    && chmod +x /start_quickstart_argilla.sh && \
```
Comment (jfcalvo) [inline quickstart.Dockerfile:34]: If I'm not wrong this is allowed right? (putting comments between lines and don't alter anything):

```suggestion
    pip3 install datasets && \
    # Install Argilla
    chmod +x /start_quickstart_argilla.sh && \
```

File: quickstart.Dockerfile [hunk 6]
```diff
@@ -1,44 +1,44 @@
...
     for wheel in /packages/*.whl; do pip install "$wheel"[server]; done && \
-    rm -rf /packages
-
-# This line add context to this image. This solution should be improved
-RUN echo -e "{  \"deployment\":  \"quickstart\" }" \
-  > /usr/local/lib/python3.9/dist-packages/argilla/server/static/deployment.json
+    rm -rf /packages && \
+    rm -rf /var/lib/apt/lists/* \
+    # This line add context to this image. This solution should be improved
+    && echo -e "{  \"deployment\":  \"quickstart\" }" \
```
Comment (jfcalvo) [inline quickstart.Dockerfile:39]: If I'm wrong ignore it:

```suggestion
    rm -rf /var/lib/apt/lists/* && \
    # This line add context to this image. This solution should be improved
    && echo -e "{  \"deployment\":  \"quickstart\" }" \
```

File: quickstart.README.md [hunk 7]
```diff
@@ -23,7 +23,7 @@
 
 
 > 🆕 🔥 Play with Argilla UI with this [live-demo](https://argilla-live-demo.hf.space) powered by Hugging Face Spaces (
-> login:`argilla`, password:`1234`)
+> login:`argilla`, password:`11223344`)
```
Comment (jfcalvo) [inline quickstart.README.md]:
```suggestion
> login:`argilla`, password:`12345678`)
```

File: quickstart.README.md [hunk 8]
```diff
@@ -113,16 +113,18 @@ This will run the latest quickstart docker image with 2 users `argilla` and `tea
 
 ### Environment Variables
 
-- `ARGILLA_API_KEY`: Argilla provides a Python library to interact with the app (read, write, and update data, log model
+- `ADMIN_USERNAME`: The admin username to login in Argilla. The default admin username is `admin`. By setting up
```
Comment (jfcalvo) [inline quickstart.README.md]:
```suggestion
- `ADMIN_USERNAME`: The admin username to log in Argilla. The default admin username is `admin`. By setting up
```

File: quickstart.README.md [hunk 9]
```diff
@@ -113,16 +113,18 @@ This will run the latest quickstart docker image with 2 users `argilla` and `tea
...
+  a custom username you can use your own username to login into the app.
+- `ADMIN_API_KEY`: Argilla provides a Python library to interact with the app (read, write, and update data, log model
   predictions, etc.). If you don't set this variable, the library and your app will use the default API key
-  i.e. `argilla.apikey`. If you want to secure your app for reading and writing data, we recommend you to set up this
+  i.e. `team.apikey`. If you want to secure your app for reading and writing data, we recommend you to set up this
```
Comment (jfcalvo) [inline quickstart.README.md]:
```suggestion
  i.e. `admin.apikey`. If you want to secure your app for reading and writing data, we recommend you to set up this
```

File: quickstart.README.md [hunk 10]
```diff
@@ -113,16 +113,18 @@ This will run the latest quickstart docker image with 2 users `argilla` and `tea
...
   variable. The API key you choose can be any string of your choice and you can check an online generator if you like.
-- `ARGILLA_PASSWORD`: This sets a custom password for login into the app with the `argilla` username. The default
-  password is `1234`. By setting up a custom password you can use your own password to login into the app.
-- `TEAM_API_KEY`: This sets the root user's API key. The API key you choose can be any string of your choice and you can
-  check an online generator if you like. The default api key is `team.apikey`.
-- `TEAM_PASSWORD`: This sets a custom password for login into the app with the `argilla` username. The default password
-  is `1234`. By setting up a custom password you can use your own password to login into the app.
+- `ADMIN_PASSWORD`: This sets a custom password for login into the app with the `argilla` username. The default
+  password is `11223344`. By setting up a custom password you can use your own password to login into the app.
```
Comment (jfcalvo) [inline quickstart.README.md]:
```suggestion
  password is `12345678`. By setting up a custom password you can use your own password to login into the app.
```

File: quickstart.README.md [hunk 11]
```diff
@@ -113,16 +113,18 @@ This will run the latest quickstart docker image with 2 users `argilla` and `tea
...
+- `ANNOTATOR_USERNAME`: The annotator username to login in Argilla. The default annotator username is `argilla`. By setting up
+  a custom username you can use your own username to login into the app.
+- `ANNOTATOR_PASSWORD`: This sets a custom password for login into the app with the `argilla` username. The default password
+  is `11223344`. By setting up a custom password you can use your own password to login into the app.
```
Comment (jfcalvo) [inline quickstart.README.md]:
```suggestion
  is `12345678`. By setting up a custom password you can use your own password to login into the app.
```

File: quickstart.Dockerfile
Comment (jfcalvo) [inline quickstart.Dockerfile:1]: Why we don't have an annotator api key?

**Review by jfcalvo — CHANGES_REQUESTED** [review]

File: quickstart.Dockerfile
Comment (frascuchon) [reply quickstart.Dockerfile:1]: Annotaror normally will work with argilla using the Argilla UI. So, less relevant to have a way to setup the annotator API key by an environment variable.

**Review by jfcalvo — APPROVED** [review]

**Pull request merged**